crates/agglayer-config/src/lib.rs

```diff
@@ -104,6 +104,10 @@ pub struct Config {
     #[serde(default = "default_prover_entrypoint")]
     #[serde(skip_serializing_if = "String::is_empty")]
     pub prover_entrypoint: String,
+
+    #[serde(default)]
+    #[serde(skip_serializing_if = "<&bool as std::ops::Not>::not")]
```

Comment:
style: I tend to write a `is_false`, which reads a bit easier ;)

Reply:
We probably don't need trace there at all.